src/main/java/Ui.java

```diff
@@ -0,0 +1,136 @@
+import java.util.ArrayList;
+
+/**
+ * Represents all the interactions Duke will have with the user.
+ */
+public class Ui {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINE = "____________________________________________"
+            + "________________\n";
+
+    /**
+     * Returns the line variable.
+     *
+     * @return line variable.
+     */
+    public static String lineGetter() {
```

Comment:
lineGetter is a noun, and we should verb for the method name.
(But I personally also like to use this kind of name. This course has soooooo many strange rules.)